examples/mnist/mnist_lib_test.py

```diff
@@ -75,9 +55,8 @@ def test_train_and_evaluate(self):
        mocked data for MNIST dataset.
     """
     # Get datasets testing-metadata directory.
-    current_dir = os.path.dirname(os.path.realpath(__file__))
-    parent_dir = os.path.dirname(current_dir)
-    data_dir = parent_dir + '/testing/datasets'
+    flax_root_dir = Path(__file__).parents[2]
```

Comment:
Add a small comment, e.g. "Go two directories up to the root of the flax repository"

Reply:
Done.